Writing Day: Doc Detective and Docs as Test (#2100)
writethedocs/www: docs/conf/portland/2024/writing-day.rst

Project List
------------

Docs as Tests & Doc Detective: Help us test your docs!
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Meet the team, get set up with tools or strategies, run some tests.**

> `Doc Detective <https://doc-detective.com/>`__ is back at Writing Day again, this time with our friends, `Docs as Tests <www.docsastests.com>`__. Does your project or product have a UI? APIs? SDKs? We can help you test them and keep your docs accurate.

Reviewer: ```suggestion
`Doc Detective <https://doc-detective.com/>`__ is back at Writing Day again, this time with our friends, `Docs as Tests <https://www.docsastests.com>`__. Does your project or product have a UI? APIs? SDKs? We can help you test them and keep your docs accurate.
```

> Docs as Tests is a tool-agnostic strategy for validating doc content by performing documented tasks on projects using automation. By validating that doc contents work as written, you can: 

Reviewer 2: ```suggestion
Docs as Tests is a tool-agnostic strategy for validating doc content by performing documented tasks directly in your products using automation. By validating that doc contents work as written, you can: 
```

* Prevent broken docs.

> * Ensure consistent UX between docs, projects, or products.

Reviewer 2: ```suggestion
* Ensure consistent UX between docs and products.
```

* Build user trust.

> Our Writing Day goal is to test at least 10 docs projects! We want every writer to be confident in their docs. 

Reviewer 2: ```suggestion
Our Writing Day goal is to test at least 10 docs sets! We want every writer to be confident in their docs. 
```

> Come by our table, and we'll help you set up Docs as Tests with whatever tools (such as Doc Detective) or strategies (like unit testing code snippets) are appropriate. Establish a zero-trust relationship between your docs and project, catching bugs in both as you go.

Reviewer 2: ```suggestion
Come by our table, and we'll help you set up Docs as Tests with whatever tools (such as Doc Detective) or strategies (like unit testing code snippets) are appropriate. Establish a zero-trust relationship between your docs and product, catching bugs in both as you go.
```
